docs/reference/modules/node.asciidoc

```diff
@@ -156,6 +156,12 @@ discovery.zen.minimum_master_nodes: 2 <1>
 ----------------------------
 <1> Defaults to `1`.
 
+Critical clusters must have at least three master-eligible nodes, with
```

Comment:
What is a "critical cluster"? What is "must"? We enforce nothing here, I want to understand where this is going, what is the reason for asserting this?

Reply:
The "critical" terminology was lifted from the 1.7 docs, but perhaps something more descriptive would be better. "Highly-available" perhaps?

"Must" is indeed a little strong - how about "should"?

We don't recommend a redundant set of master-eligible nodes (i.e. ≥ 3) anywhere in the docs as far as I can tell, but I feel that should be written down somewhere.